### PR TITLE
Fix invite lists, only return invites you have received

### PR DESCRIPTION
**Pull request by koutst**
- The list view on invites currently returns all invites. These changes limit those invites to the ones you have received.

### CODE REVIEW

**Review by yattias — APPROVED** [review]
LGTM

**Review by gzurowski — APPROVED** [review]
LGTM 👍

File: src/invite/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            password="password",
+            email="user1@researchhub_test.com",
+        )
+        self.recipient = get_user_model().objects.create_user(
+            username="user2@researchhub_test.com",
+            password="password",
+            email="user2@researchhub_test.com",
+        )
+
+        # Create org
```
Comment (koutst) [inline src/invite/tests/test_views.py:75]:
```suggestion
        # Create note
```

File: src/invite/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        )
+        self.recipient = get_user_model().objects.create_user(
+            username="user2@researchhub_test.com",
+            password="password",
+            email="user2@researchhub_test.com",
+        )
+
+        # Create note
+        self.client.force_authenticate(self.sender)
+        response = self.client.post("/api/note/", {"name": "ORG A"})
```
Comment (koutst) [inline src/invite/tests/test_views.py:77]:
```suggestion
        response = self.client.post("/api/note/", {"name": "NOTE A"})
```

**Pull request merged**